examples/box/Vagrantfile

```diff
@@ -20,6 +20,7 @@ Vagrant.configure("2") do |config|
     vmck.vmck_url = ENV['VMCK_URL']
     vmck.memory = 512
     vmck.cpus = 1
+    vmck.restrict_network = true
```

Comment:
Do we really want this to be the default?

Reply:
We can leave it to `false`. I just want to have the option there.

Reply:
If it's just for documentation you can say `= false`.